src/main/java/frc/robot/util/AutoGenerator.java

```diff
@@ -79,76 +99,41 @@ public AutoGenerator(DrivetrainSubsystem drivetrainSubsystem, FloorArmSubsystem
+        autoTab.addString("Current Starting Action: ", () -> getSelectedStartAction()).withSize(2, 1).withPosition(0, 2);
+        autoTab.add("Current Auto Trajectory", field).withSize(6, 4).withPosition(2, 1).withProperties(Map.of("colors", "yellow"));
     }
 
-    public Command getExitAndBalance(StartLocation startArea) {
-        if(startArea == StartLocation.ONE || startArea == StartLocation.TWO) {
-            PathPlannerTrajectory path = PathPlanner.loadPath("ExitAndBalance A" + startArea.value, defaulPathConstraints);
-            DriverStation.reportWarning("Auto Area: " + startArea.value, false);
-            return builder.fullAuto(path);
-        }else {
-            return null;
-        }
+    private String getSelectedPath() {
+        return pathActionSelector.getSelected().value + locationSelector.getSelected().value;
     }
 
-    public Command getScoreConeAndExit(StartLocation startArea) {
-        if(startArea == StartLocation.ONE || startArea == StartLocation.TWO) {
-            PathPlannerTrajectory path = PathPlanner.loadPath("ScoreConeAndExit A" + startArea.value, defaulPathConstraints);
-            DriverStation.reportWarning("Auto Area: " + startArea.value, false);
-            return builder.fullAuto(path);
-        }else {
-            return null;
-        }
+    public Command getAutoCommand() {
+        return builder.fullAuto(PathPlanner.loadPath(getSelectedPath(), defaulPathConstraints));
     }
 
-    public Command getScoreCubeAndExit(StartLocation startArea) {
-        if(startArea == StartLocation.ONE || startArea == StartLocation.TWO) {
-            PathPlannerTrajectory path = PathPlanner.loadPath("ScoreCubeAndExit A" + startArea.value, defaulPathConstraints);
-            DriverStation.reportWarning("Auto Area: " + startArea.value, false);
-            return builder.fullAuto(path);
-        }else {
-            return null;
-        }
+    public String getSelectedStartAction() {
+        eventMap.put("startAction", startActionMap.get(startActionSelector.getSelected().value));
+        return startActionSelector.getSelected().value;
     }
 
-    public Command getScoreLowAndExit(StartLocation startArea) {
-        if(startArea == StartLocation.ONE || startArea == StartLocation.TWO || startArea == StartLocation.THREE) {
-            PathPlannerTrajectory path = PathPlanner.loadPath("ScoreLowAndExit A" + startArea.value, defaulPathConstraints);
-            DriverStation.reportWarning("Auto Area: " + startArea.value, false);
-            return builder.fullAuto(path);
-        }else {
-            return null;
-        }
+    /* Called periodically while disabled */
+    public void updateSelectorPose() {
+        PathPlannerTrajectory path = PathPlanner.loadPath(getSelectedPath(), defaulPathConstraints);
+        field.setRobotPose(path.getInitialPose());
+        field.getObject("traj").setTrajectory(path);
```

Comment:
it would be cool to set this to our "drive" Field2d widget as that would centralize to a single Field2d view of our auto and in match pose

Reply:
Thought about it and talked to Noelle and we decided it would be more clear if these remained separate.